### PR TITLE
add exitCodes

### PR DESCRIPTION
**Pull request by alirezatabatabaeian**

### CODE REVIEW

**Comment by alirezatabatabaeian** [comment]
I've synced my repo and made a new PR

**Comment by anton-trunov** [comment]
@novusnota could you suggest where this page should go with the new docs structure?

**Comment by novusnota** [comment]
I think that the current place (Language→Reference) is fine for this.

All there's to do from @alirezatabatabaeian side is to:
1. Stop including `yarn.lock` in the commit. You don't have to remove it from the repository, just don't include it unless there were important updates related to dependencies in `package.json`
2. Use `_meta.js` over `_meta.json`.

**Comment by alirezatabatabaeian** [comment]
Hi dear @novusnota 
After a check, I've found out many things has been change in this repo so I guess I have to sync my fork and make another PR.
Right?

**Comment by alirezatabatabaeian** [comment]
Even after deleting yarn.lock from my repo, it still has conflict. why is that?

File: yarn.lock
Comment (anton-trunov) [inline yarn.lock:1]: you need to remove the changes to `yarn.lock` from your branch as opposed to removing it from the repository

**Comment by novusnota** [comment]
Hey, @alirezatabatabaeian, I've added a simple stub for your page in the recent commit, here it is — https://docs.tact-lang.org/language/ref/exit-codes.

Now, all you've got to do, is:
1. Save your copy of that page somewhere
2. Close this PR
3. Clone the repo once again
4. Add your copy of `exit-codes.mdx`, overriding it's contents
5. Check, that everything's working
6. Commit ONLY the `exit-codes.mdx` page

As @anton-trunov said, you don't need to remove `yarn.lock` from the repository, just don't include it in a commit. Simply do just `git add pages/language/ref/exit-codes.mdx` once you're done, then commit it and push it to your fork, then create a PR back here.

Quick tip/suggestion for your future work:
* If working from CLI, use tools like [lazygit](https://github.com/jesseduffield/lazygit).
* If working from VSCode, use it's Git tab for inclusion/exclusion of files to the commit (note, that this doesn't mean addition/deletion of the files on your hard drive).

**Comment by reveloper** [comment]
@alirezatabatabaeian, Thank you for the excellent PR to the Tact documentation! Having reviewed the current page, I have the following ideas for further improvements:

1. The examples with Exit Codes are very useful, but we know that most people are primarily looking for short explanation. We could start with a table, and include at least those that are mentioned in the article, similar to (https://docs.ton.org/learn/tvm-instructions/tvm-exit-codes), but Tact context related description appreciated. 
In this section, under the table (Exit Codes) as a TIP, briefly and clearly write about how the Tact Compiler collects all exit codes in a *.md file and you can always conveniently track them in the directory along the path ... .

2. Structure the section with code examples a bit more detailed:
-  1 Exit Codes
-  2 Exit Code Examples
-- 2.1 Compute Phase Exit Codes
-- 2.2 Action Phase Exit Codes
-- 2.3 Tact Compiler Exit Codes

3. I noticed that the examples and the description mention errors directly related to TVM; it would be great to completely avoid mentioning TVM in the main description and think on the level of Tact abstractions. It seems logical not to show examples with impossible cases here, though keeping them in the Fireworks contract is okay.

Please remove from the text exclamations that do not match the style of documentation:
https://github.com/alirezatabatabaeian/tact-docs/commit/ffab5affc905867fcd68b9933e23f47e2d2a748d#diff-cd0c77e17d192b080e48d3df644895e8710f6fffdf6a518f4f2751b406e5c44dR39

https://github.com/alirezatabatabaeian/tact-docs/commit/ffab5affc905867fcd68b9933e23f47e2d2a748d#diff-cd0c77e17d192b080e48d3df644895e8710f6fffdf6a518f4f2751b406e5c44dR83

The task of this page is to deliver precise information as quickly as possible, so any distractions here hinder the main goal.

cc @anton-trunov @novusnota

**Pull request closed without merging**